prisma/migrations/20240802000848_image_migration_aug_1_2024/migration.sql

```diff
@@ -0,0 +1,72 @@
+-- CreateEnum
```

Comment:
@Kholoudxs55kh PLease do NOT NOT NOT EVER include migrations in important hotfix :-)

migrations can potentially corrupt data, they cannot be rolled back, they should ALWAYS be in SEPARATE PRS

THANK YOU

Reply:
@gvelez17 Removed Them